docs/pages/spacing.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Spacing
+name: Spacing
+path: /spacing
+---
+
+import Spacing from '../components/Spacing'
+
+### Spaces
+
+Use space tokens to define padding or margin when designing a component or layout.
+
+<Spacing />
```

Comment:
I apologize that I didn't do a very good job of documenting this ticket, I had some ideas in my head that I didn't fully flesh out here as I was expecting myself to pick it up eventually.

We don't need a 1:1 mapping of what's in UXPin to what's here in cauldron but rather something more fleshed out. Here's an example of something that is a little more useful:

- https://designsystem.digital.gov/design-tokens/spacing-units/